### PR TITLE
Add initial tests

### LINKED ISSUE #18: Tests
We should set up some sort of testing infrastructure to make sure things don't break whenever we make changes :)

### PR DESCRIPTION
**Pull request by arthuralee**
fixes #18

### CODE REVIEW

File: src/cli.js [hunk 1]
```diff
@@ -58,16 +59,15 @@ const connect = (options) => {
 
     if (!isTaken) {
       const child = fork(
-        require.resolve('./startServer.js'), args, {/* silent: true */}
+        require.resolve('./startServer.js'), args, { silent: true }
```
Comment (arthuralee) [inline src/cli.js]: This was the reason things weren't working at first. Everything seems to work with this turned on. Is this fine?

Comment (akleiner2) [inline src/cli.js]: Yeah, that should be fine

File: src/cliUtils.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export function clearLine() {
```
Comment (akleiner2) [inline src/cliUtils.js]: Can you ES6-ify this? `export const clearLine = () => {...}`

Comment (arthuralee) [inline src/cliUtils.js]: i'd say this is ES6, and using function syntax makes for better stack traces

Comment (akleiner2) [inline src/cliUtils.js]: ah yeah, that's fair

**Pull request merged**